orchagent/portsorch.cpp

```diff
@@ -1179,15 +1251,25 @@ bool PortsOrch::addBridgePort(Port &port)
         return false;
     }
 
+    m_portList[port.m_alias] = port;
     SWSS_LOG_NOTICE("Add bridge port %s to default 1Q bridge", port.m_alias.c_str());
 
     return true;
 }
 
-bool PortsOrch::removeBridgePort(Port port)
+bool PortsOrch::removeBridgePort(Port &port)
```

Comment:
>BridgePo [](start = 22, length = 8)

symmetric to addBridgePort, we can first check if the port is already in bridge or not, if the port is not in bridge, we can return true.

Reply:
there is back and forth for addBridgePort() and removeBridgePort().  
Hope the following implementation is ok:
"Calling addBridgePort() only when bridge port not created for port/lag, and calling removeBridgePort() only when port/lag not in any VLAN"

Reply:
there is no back and forth, what I meant is that

for addBridgePort() if it is already a bridge port, you can return true. For removeBridgePort, if it is not a bridge, you can also return true.

they are different than when the port is still in vlan, you should return false.

Reply:
Ok, let me change it again, hope the new code will match what you want.